### PR TITLE
Refactor ec2_import_image* modules

### PR DESCRIPTION
**Pull request by abikouo**
##### SUMMARY
Refactor ec2_import_image,ec2_import_image_info modules

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

ec2_import_image ec2_import_image_info

### CODE REVIEW

File: plugins/modules/ec2_import_image.py [hunk 1]
```diff
@@ -16,8 +16,8 @@
 options:
   state:
     description:
-      - Use O(state=present) to import single or multi-volume disk images or EBS snapshots into an Amazon Machine Image (AMI).
-      - Use O(state=absent) to cancel an in-process import virtual machine task.
+      - Use I(state=present) to import single or multi-volume disk images or EBS snapshots into an Amazon Machine Image (AMI).
+      - Use I(state=absent) to cancel an in-process import virtual machine task.
```
Comment (alinabuzachis) [inline plugins/modules/ec2_import_image.py:20]:
```suggestion
      - Use O(state=present) to import single or multi-volume disk images or EBS snapshots into an Amazon Machine Image (AMI).
      - Use O(state=absent) to cancel an in-process import virtual machine task.
```

File: plugins/modules/ec2_import_image.py [hunk 2]
```diff
@@ -101,7 +101,7 @@
   encrypted:
     description:
       - Specifies whether the destination AMI of the imported image should be encrypted.
-      - The default KMS key for EBS is used unless you specify a non-default KMS key using O(kms_key_id).
+      - The default KMS key for EBS is used unless you specify a non-default KMS key using I(kms_key_id).
```
Comment (alinabuzachis) [inline plugins/modules/ec2_import_image.py:104]:
```suggestion
      - The default KMS key for EBS is used unless you specify a non-default KMS key using O(kms_key_id).
```

File: plugins/modules/ec2_import_image.py [hunk 3]
```diff
@@ -113,7 +113,7 @@
       - An identifier for the symmetric KMS key to use when creating the encrypted AMI.
         This parameter is only required if you want to use a non-default KMS key;
         if this parameter is not specified, the default KMS key for EBS is used.
-        If a O(kms_key_id) is specified, the O(encrypted) flag must also be set.
+        If a I(kms_key_id) is specified, the I(encrypted) flag must also be set.
```
Comment (alinabuzachis) [inline plugins/modules/ec2_import_image.py:116]:
```suggestion
        If a O(kms_key_id) is specified, the O(encrypted) flag must also be set.
```

**Review by alinabuzachis — APPROVED** [review]

File: plugins/modules/ec2_import_image_info.py [hunk 4]
```diff
@@ -190,17 +175,18 @@ def main():
...
-    client = module.client("ec2", retry_decorator=AWSRetry.jittered_backoff())
+    client = module.client("ec2")
     params = {}
 
     if module.params.get("filters"):
         params["Filters"] = module.params["filters"]
     if module.params.get("import_task_ids"):
         params["ImportTaskIds"] = module.params["import_task_ids"]
 
-    import_image_info = helper_describe_import_image_tasks(client, module, **params)
```
Comment (hakbailey) [inline plugins/modules/ec2_import_image_info.py:201]: Can this `helper_describe_import_image_tasks()` function also be deleted from module_utils/ec2.py?

File: plugins/modules/ec2_import_image.py [hunk 5]
```diff
@@ -498,7 +497,7 @@ def main():
     state = module.params.get("state")
 
     try:
-        client = module.client("ec2", retry_decorator=AWSRetry.jittered_backoff())
+        client = module.client("ec2")
```
Comment (hakbailey) [inline plugins/modules/ec2_import_image.py:500]: I think we can remove this try/except block per the discussion [here](https://github.com/ansible-collections/amazon.aws/pull/2138#discussion_r1660951340).

**Review by hakbailey — APPROVED** [review]
A couple of minor suggestions but this looks good!

**Pull request merged**